js/renovation/ui/grids/data_grid/common/data_grid_view_props.ts

```diff
@@ -4,4 +4,6 @@ import { GridInstance } from './types';
 @ComponentBindings()
 export class DataGridViewProps {
   @OneWay() instance!: GridInstance;
+
+  @OneWay() showBorders!: boolean;
```

Comment:
Re-export showBorders like in https://github.com/MikeVitik/DevExtreme/blob/27500a0cf0f8a74d9196e9675efb7d4fbcc53dd9/js/renovation/ui/pager/pages/large.tsx#L93